federatedscope/gfl/trainer/graphtrainer.py

```diff
@@ -11,7 +11,11 @@ class GraphMiniBatchTrainer(GeneralTorchTrainer):
     def _hook_on_batch_forward(self, ctx):
         batch = ctx.data_batch.to(ctx.device)
         pred = ctx.model(batch)
-        label = batch.y.squeeze(-1).long()
+        # TODO: deal with the type of data within the dataloader or dataset
+        if 'regression' in ctx.cfg.model.task.lower():
+            label = batch.y
+        else:
+            label = batch.y.squeeze(-1).long()
```

Comment:
this is necessary as some datasets have `batch.y.shape` to be (batch_size, 1), right?

Reply:
@rayrayraykk , I'm not sure if it's appropriate to remove `squeeze(-1)`